content/en/docs/tutorials/security/apparmor.md

```diff
@@ -44,17 +44,17 @@ applications and cluster from other angles as well.
 Make sure:
 
 1. Kubernetes version is at least v1.4 -- Kubernetes support for AppArmor was added in
-   v1.4. Kubernetes components older than v1.4 are not aware of the new AppArmor annotations, and
-   will **silently ignore** any AppArmor settings that are provided. To ensure that your Pods are
+   v1.4. Kubernetes components older than v1.4 do not support the AppArmor API, and
+   will **reject** any worloads with AppArmor settings provided. To ensure that your Pods are
```

Comment:
1.4 was a pre-history thing. We can safely remove the related text.

```suggestion
   v1.4. To ensure that your Pods are
```

Reply:
We don't even need to ask readers to ensure they are running Kubernetes 1.4 or later.

Better is to use a `{{< version-check >}}` shortcode, set `min-kubernetes-server-version` in the front matter, _and_ provide some English text to explain what readers should do if their cluster is older than v1.26

(We can drop that additional text once v1.30 is out).

Reply:
Added the `version-check` and surrounding wording as well as the `min-kubernetes-server-version` == 1.26 metadata in [d6abe42](https://github.com/kubernetes/website/pull/37191/commits/d6abe421da86cda4632870d5f551d9fd2b4a1654)